packages/fxa-auth-server/lib/senders/emails/templates/verifyShortCode/en.ftl

```diff
@@ -1,6 +1,7 @@
 # Variables:
 #  $code (Number) - e.g. 123456
-verifyShortCode-subject-2 = Confirmation code: { $code }
-verifyShortCode-title = Is this you signing up?
-verifyShortCode-prompt-2 = If yes, use this confirmation code in your registration form:
+verifyShortCode-subject-3 = Confirm your account
+verifyShortCode-title-2 = Open the internet with { -brand-firefox }
+verifyShortCode-title-subtext = Confirm your account and get the most out of Firefox everywhere you sign in starting with:
```

Comment:
```suggestion
# Information on the browser and device triggering this confirmation email follows below this string.
verifyShortCode-title-subtext = Confirm your account and get the most out of { -brand-firefox } everywhere you sign in starting with:
```
Suggest adding a comment about what will displayed after "starting with:" for context.
Please use brand placeholder here too.